### ISSUE TITLE
Use Suru blue as the highlight color instead of orange

### ISSUE THREAD

**Issue by kq01526** [Ubuntu/gnome-shell-communitheme#54]
Hi,

In a a lot of themes today, blue is being used as the highlight color. This is easy on the eyes and looks good.

The Ubuntu theme on the other hand uses bright orange for highlights. This is not easy on the eyes and does not look good (of course the latter can be subjective).

Why can't we change that to the official Suru blue instead, which would be much easier on the eyes than orange, see:

https://design.ubuntu.com/brand/colour-palette/

?

**Comment by kq01526** [comment]
Also requested for Suru icons: https://github.com/snwh/suru-icon-theme/issues/27

**Comment by galgalesh** [comment]
I also think we should give that color a try and see if it also works on the dark backgrounds.

**Comment by ya-d** [comment]
**TL;DR**
* Global decision on colors needed: which ones to use, what semantics they have and where to use which. Stay coherent and consistent!
* If folders and highlights get purple shouldn't sliders, spinner, links and so on do as well? Or do you really want orange, aubergine *and* blue (plus green and red)?
* Possible short term/easier approach: Keep orange where it is but tone it done for text selection and folders. Than decide for secondary color (purple or bright blue, maybe darker blue).

This should really be a general discussion about use of colors in all parts (>>  gtk) of the theme. I'm absolutely in favor of the latest commit which uses the same (blue) color for all progress bars (instead of some being green, some being blue). I don't think *adding* another color is the right way; it should than replace the suru blue (or (parts of the) orange as proposed here – or both of them). A combination of two colors plus green and red seems more than enough to me. Also consider that "neutral messages" currently are purple. 

Personally I think neither sidebar selection nor focus indicators are fundamentally distracting (also see linked Canonical blog: hero item + focus). I can see it being an issue for text selection and (less so) for folder icons as they can result in quite a lot of orange. Therefor a more approachable solution could be to keep orange as primary (focus/attention/this-is-where-you-are/branding) color and tone both of them down, e.g. by using a lighter hue of orange (matching Suru folders) for text selection and maybe using white/grey or very light orange (see bottom right corner of Suru folders) for the "inner side" of folders as has been the case with Ambiance. (Purple could than still be used as secondary color instead of blue being both calming and warm.)

**Comment by galgalesh** [comment]
Yes, the proposal is that aubergine replaces the Suru blue + the orange for everything "focus", "active" and "selection"-like.

**Comment by NusiNusi** [comment]
Are these the suru-colours? Could these be a starting point?
https://docs.google.com/presentation/d/1FtHFW67ycl6uvxZqKIZwyinVyOuV9NxXDuJv1SwQ70k/edit#slide=id.g13b3c671fc_1_0

**Comment by galgalesh** [comment]
For how the Suru blue/purple works with the dark theme:

The Suru blue seems to work pretty well with a dark theme: 
![image](https://user-images.githubusercontent.com/1492981/36218252-0cc476fe-11b4-11e8-8e16-8bf12bdb3999.png)

Someone on the community forum is creating an Adwaita + Suru colors theme: https://community.ubuntu.com/t/adwaita-theme-with-unity8-colours-and-other-variants/4041/13?u=merlijn-sebrechts

I personally really like those colors.

**Comment by kq01526** [comment]
> @galgalesh wrote:
> The Suru blue seems to work pretty well with a dark background

And what about aubergine?

By the way, good to see that my idea seems to be considered uniformly by most people (I was actually expecting to get some backlash for it or at least have people divided into two camps, one voting for aubergine/blue, one voting to keep orange) 🙂 .

But what about the folder icons? I have nobody seen mentioning them.

Honestly, the orange folder icons just look hideous, they have always been one of the main reasons I would never consider using the standard Ubuntu theme.

If this could finally be changed for Ubuntu 18.04, that would be great.

**Comment by galgalesh** [comment]
@kq01526 

Folder icons are part of the icon theme. I see you've created [an issue on that repo](https://github.com/snwh/suru-icon-theme/issues/27) there is not much use discussing it here since it's a different designer/developer.

**Comment by madsrh** [comment]
I believe we did try this early in the process, because the orange e.g in the sidebar in Nautilus might feel heavy, but as mentioned it's really lacks branding without. 
I can't find a single mockup that I'll illustrates this :( 

I really like the current design as one orange "hero" item is allowed, this doesn't _hurt my eyes_ / steal my attention. So should we create new mockups to give this a try or simply create a branch with these colors? @Luxamman @godlyranchdressing ?

Regarding the icons, there's also a less heavy (I'm looking for a better word)[ design coming](https://github.com/Ubuntu/gtk-communitheme/issues/43).

**Comment by kq01526** [comment]
> @madsrh wrote:
> Regarding the icons, there's also a less heavy (I'm looking for a better word) [design coming](https://github.com/Ubuntu/gtk-communitheme/issues/43).

Still looks hideous. I know this might sound like trolling or overexaggerating, but seriously, those warm orange tones induce a feeling of having to puke inside of me.

!!! This is not a joke or trolling !!!

Seriously, I am getting some kind of nausea in the stomach by looking at those orange folder icons (because of the color).

I know this must sound ridiculous, I myself feel like this is kinda ridiculous while writing it, but it's just the way it is.

Aubergine folder icons sure would be a heck of a lot better (I assume, I would need to see it first of course).

Or just plain yellow or beige folder icons would also be better (like in Windows 10 or Adwaita).

**Comment by Luxamman** [comment]
Maybe @kq01526 needs a new display with a better color reproduction^^ 
just joking ;)

Like @madsrh said, we had those discussions a lot of times now and if it is a big problem, you maybe should use vanilla Gnome (also for your stomach if it is true what you write). Because the colors are of course subjective and three different people have three different opinions. 

The orange is for years now part of the branding and is not telling the user anymore that this is something dangerous. I'm sorry that you have such a big problem with the color, we will think again about it, but this is a decision which goes far beyond one's single opinion. 

This is like telling the VLC guy's that the orange cone icon is ugly. Which it is, kind of. But did they change it till now? And why not? Branding is important and should not be ignored in a world full of imitations. And Ubuntu is orange (not aubergine, this color has no further meaning for the user), which also should not be ignored.

**Comment by NusiNusi** [comment]
@kq01526 The Communitheme tries to build around the Suru-Icons which were developed for Unity 8. The orange might look very aggressive but it is somehow a brand-color. At the moment i really think that the orange looks different on different displays. There are displays (maybe yours) that have very strong colours (my old samsung notebook has such a display) and on these the orange might look a bit too aggressive. On my newer dell-notebook (which has a mat display) the orange looks okay. This might be a stupid theory but i think most ubuntu-users (especially professional users) have a professional notebook with a mat display so they can work with it even when the sun is shining. Thats why they dont see the orange as "aggressive" as you are or me on my older notebook. Just a theory.

**Comment by galgalesh** [comment]
@luxamman

I think that the Canonical designers dropping Orange is a good indicator that we shouldn't dismiss the issues with orange so lightly. This is clearly not a single user's preference.

The UX research was clear and that's not surprising: blue is so entrenched as the Color of selection. It's the case on any os, it's on github, all electron apps, almost every website, the majority of themes on Linux, the default HTML stylesheet..

I'm all for orange, we obviously need to keep it because it creates the brand, but why should it be the default Color for selection? I don't think "Ambiance was like that" is a good enough reason to ignore Canonical's design. I think everyone is glad that Ubuntu isn't brown anymore.

This is where you come in, @luxamman, how can we integrate orange in the design in a way that matches what semantic value users give to colors, conforms to industry standards more, and is less loud?

As a strawman: what about using orange only to indicate "location"?

- location = orange so stackswitcher,  the current folder in nautilus sidebar and current pane in the tweaks sidebar.

- selection and activeness = Suru blue, so that's stuff like text selection and sound level is "selection and activeness".

I'm on mobile so I can't go into detail much, but let's start a discussion about this?

**Comment by kq01526** [comment]
> @Luxamman wrote:
> Ubuntu is orange (not aubergine, this color has no further meaning for the user)

Oh, really? When I boot Ubuntu, I see this:

![https://i.stack.imgur.com/CaJBs.png](https://i.stack.imgur.com/CaJBs.png)

Don't know about you, but looks massively aubergine to me... 🙂

**Comment by Luxamman** [comment]
Well, that is something over the topic. 
This is a background of the start loading screen (and the terminal) and this has no further meaning for UX (because there is no interaction, whatever) and is to the people more a reddish dark than aubergine - even if the ground color maybe is (yes, I tested this before^^). 

---

But before this discussion is getting an "I want to be right" discussion, I like to have reasonable and logic, also meaningful opinions and possibilities. 

- Aubergine has no meaning in the UX because it's never been used and the user is therefore not used to it. Also, aubergine is the Canonical brand color, which is used for "consumer or enterprise focus". This is not excluding it, but also not favor it. But it would mean another strong (and very present) color and people have to get used to it.
- Blue is, of course, the selection color of choice for a lot of systems and themes. People are used to it. It is a quite neutral color and works also quite well with the orange. 
- A color change would mean, that the whole system should adapt it to be consistent. And that would mean to kill the orange in a lot more places - because "selection" is everywhere (like @ya-d pointed out). Also, we use the color now for indicators and we would lose a lot of "Ubuntu" style. 

So if we change the selection color only in "Files" - wouldn't we have another discussion about inconsistency? Or maybe it is too much like vanilla Gnome? Not enough Ubuntu?

Also, @kq01526 is talking about the icon theme ("orange folder icons") and this has nothing to do with selection color. Those are icons. The icons of Ubuntu are orange since... forever. 

In conclusion, this discussion is talking about branding, the icon theme, and the selection color. 
The selection of folders and files is not very eye pleasing in any color (guess this is a Gnome thing, is discussed and maybe will be fixed somehow) and also should be said: Unity 8 wasn't finished on the desktop. 

For me, there is another internal discussion on the table (and maybe mockups and polls) to find a consensus. 
Even if the best consensus for me would be a color-switch in the settings for the selection color - orange, blue, aubergine and whatever. You choose, problem solved ;)

**Comment by galgalesh** [comment]
> Aubergine

I thought we were all talking about the same color (Suru blue), since that used to be more aubergine-ish. It doesn't help that I have a hard time seeing the difference between dark blue and purple. :wink: 

I agree to not use aubergine, it's part of the Canonical enterprise brand, not Ubuntu, so it's basically out of the question. I've changed the title of this issue. :+1: 

> So if we change the selection color only in "Files" - wouldn't we have another discussion about inconsistency? Or maybe it is too much like vanilla Gnome? Not enough Ubuntu?

I'm obviously talking about changing the selection color everywhere, I'm sorry if that wasn't clear. Changing the selection color everywhere would provide more consistency, since a number of (electron) apps and websites today use blue as selection color, regardless of what the OS theme actually uses.

> this discussion is talking about branding, the icon theme, and the selection color.

So let's narrow down the discussion. The discussion about the icon theme has already been redirected to the icon theme repo. I removed the mention of Aubergine from the title and the issue. So this is a discussion about the selection color. The discussion about branding should be held at the same time, though, since the primary argument for not switching the selection color to blue seems to be "branding".

**Comment by pojntfx** [comment]
In case you guys are interested: As already pointed out before I am currently creating different color variants of Adwaita with the colors from [Unity8 Visual Style](https://docs.google.com/document/d/1CjPvxz_dk9Cn5HiZ_d0POYtqNAHjO1c5omAbNjZJiKg/edit#heading=h.33c593elo5ur) here: [Adwaita-Ubuntu](https://github.com/pojntfx/gtk-adwaita-ubuntu). I've just created a orange version alongside 2 blue versions in case you want to try it out which one works best for communitheme/yourself (I know, it's adwaita and not the communitheme base, but the colors should give a impression).

**Comment by kq01526** [comment]
> @Luxamman wrote:
> This is a background of the start loading screen (and the terminal) and this has no further meaning for UX (because there is no interaction, whatever) ***and is to the people more a reddish dark than aubergine*** - even if the ground color maybe is (yes, I tested this before^^).

LOL, and you are telling me I should get a more color accurate monitor? On even the crappiest monitor out there, it is obvious that the Plymouth theme has a aubergine background and not a red background.

> @Luxamman wrote:
> Aubergine has no meaning in the UX because it's never been used and the user is therefore not used to it. Also, aubergine is the Canonical brand color, which is used for "consumer or enterprise focus". This is not excluding it, but also not favor it. But it would mean another strong (and very present) color and people have to get used to it.

LOL, sorry, but I really have a hard time to take you serious now when even one of your own Ubuntu webpages is half aubergine and half orange, but see for yourself:

http://ubuntu.luxam.at/

🙂 

Whereas another of your webpages shows an entirely aubergine UI plus a wallpaper that is aubergine in large areas, see:

http://stefaneduardkrenn.com/finding-ubuntu/

🙂 

What you are writing here does not match up with the reality. You make it sound as if aubergine is completely absent from Ubuntu even though it clearly is not.

> @Luxamman wrote:
> I agree to not use aubergine, it's part of the Canonical enterprise brand, ***not Ubuntu***

Yeah, right, see above... 🙄 

PS: I have nothing against preferring blue over aubergine (I myself do), but coming here and stating that aubergine is not part of Ubuntu branding/theming really is quite hilarious...

I mean, just take a look at the link that I already posted in the original post:

https://design.ubuntu.com/brand/colour-palette/

Can it be any more obvious?

**Comment by galgalesh** [comment]
Ok, it seems that this issue has derailed beyond hope. Closing this now, I propose to revisit this topic after a cooling-off period.The forum is probably a better place to have free-form discussion like this.
